


 10%|██████████▋                                                                                                    | 30/313 [00:07<01:11,  3.94it/s]
Traceback (most recent call last):
  File "/Users/pratikkadlak/Documents/Convolution_Neural_Network_a2/PartA/train_PartA.py", line 653, in <module>
    X_train, y_train = read_images(train_path)
                       ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/pratikkadlak/Documents/Convolution_Neural_Network_a2/PartA/train_PartA.py", line 41, in read_images
    for image, label in tqdm(data):
  File "/Users/pratikkadlak/anaconda3/lib/python3.11/site-packages/tqdm/std.py", line 1178, in __iter__
    for obj in iterable:
  File "/Users/pratikkadlak/anaconda3/lib/python3.11/site-packages/torch/utils/data/dataloader.py", line 630, in __next__
    data = self._next_data()
           ^^^^^^^^^^^^^^^^^
  File "/Users/pratikkadlak/anaconda3/lib/python3.11/site-packages/torch/utils/data/dataloader.py", line 674, in _next_data
    data = self._dataset_fetcher.fetch(index)  # may raise StopIteration
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/pratikkadlak/anaconda3/lib/python3.11/site-packages/torch/utils/data/_utils/fetch.py", line 51, in fetch
    data = [self.dataset[idx] for idx in possibly_batched_index]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/pratikkadlak/anaconda3/lib/python3.11/site-packages/torch/utils/data/_utils/fetch.py", line 51, in <listcomp>
    data = [self.dataset[idx] for idx in possibly_batched_index]
            ~~~~~~~~~~~~^^^^^
  File "/Users/pratikkadlak/anaconda3/lib/python3.11/site-packages/torchvision/datasets/folder.py", line 229, in __getitem__
    sample = self.loader(path)
             ^^^^^^^^^^^^^^^^^
  File "/Users/pratikkadlak/anaconda3/lib/python3.11/site-packages/torchvision/datasets/folder.py", line 268, in default_loader
    return pil_loader(path)
           ^^^^^^^^^^^^^^^^
  File "/Users/pratikkadlak/anaconda3/lib/python3.11/site-packages/torchvision/datasets/folder.py", line 248, in pil_loader
    return img.convert("RGB")
           ^^^^^^^^^^^^^^^^^^
  File "/Users/pratikkadlak/anaconda3/lib/python3.11/site-packages/PIL/Image.py", line 911, in convert
    self.load()
  File "/Users/pratikkadlak/anaconda3/lib/python3.11/site-packages/PIL/ImageFile.py", line 269, in load
    n, err_code = decoder.decode(b)
                  ^^^^^^^^^^^^^^^^^
KeyboardInterrupt